### PR TITLE
Feature/js questioncard

### PR DESCRIPTION
**Pull request by vaupunkt**
I created the javascript-functionality for the "Show-Answer"-Button

### CODE REVIEW

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	padding: 15px;
+	margin: 15px auto 15px auto;
+}
+
+.button--showAnswer {
+	background-color: #a3bbad;
+	color: #312509;
+}
+
+.button--showAnswer:hover {
```
Comment (josch87) [inline components/button/button.css:15]: You're using two dashes camelCase BEM style, nice!
https://en.bem.info/methodology/naming-convention/#camelcase-style

File: javascript/script.js
Comment (josch87) [inline javascript/script.js:1]: This only hiddes the answer in the frontend using CSS but people can still cheat looking it up in the source code. You could think about using the createElement() method (https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement?retiredLocale=de) instead (and later on get the answer from the MongoDB using AJAX).

File: components/questioncard/questioncard.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	color: var(--light-font-color);
+	font-size: 0.9rem;
+}
+
+.questionViewPort {
+	height: 100vh;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	scroll-snap-align: center;
```
Comment (josch87) [inline components/questioncard/questioncard.css:12]: NICE!

File: userprofile.html [hunk 3]
```diff
@@ -4,41 +4,42 @@
...
 						</section>
-						<section class="bookmark-counter counterlabel">
-							10 Bookmarks
+						<section class="counter__label">
+							<object class="counter__label__number">10</object> Bookmarks
 						</section>
 					</section>
 				</article>
 			</section>
 			<section class="settings">
```
Comment (josch87) [inline userprofile.html:39]: settings are very close to the footer.

**Review by josch87 — APPROVED** [review]
Looks very good!

File: javascript/script.js
Comment (vaupunkt) [reply javascript/script.js:1]: that's true and it is something to definetly add in the future! :)

**Pull request merged**